src/test/java/seedu/address/logic/commands/CreatePastAppointmentCommandTest.java

```diff
@@ -44,4 +48,15 @@ public void execute_allFieldsSpecifiedFilteredList_success() {
         // remove the appointment from the model for tear down
         expectedModel.getFilteredPersonList().get(0).getPastAppointments().remove(pastAppointment);
     }
+
+    @Test
+    public void execute_invalidIndex_throwsCommandException() {
+        Set<Medication> medication = new HashSet<>();
+        medication.add(new Medication("Paracetamol"));
+        PastAppointment pastAppointment = new PastAppointment(LocalDate.now(), medication, "Fever");
```

Comment:
I believe for code quality, you are able to abstract out the preparation of a `pastAppointment` to be a private method. For reference, refer to the `preparePredicate()` method from the other `CommandTest` files.

Reply:
Rejecting the creation of the method as it would only serve 2 test cases. However, I have replaced this with a common `PastAppointment` instance returned by a `PastAppointmentBuilder`.